sentry-android-core/src/main/java/io/sentry/android/core/performance/AppStartMetrics.java

```diff
@@ -354,4 +354,26 @@ public static void onContentProviderPostCreate(final @NotNull ContentProvider co
       measurement.setStoppedAt(now);
     }
   }
+
+  /**
+   * @param hasBundle true if the activity onCreate had a non-null bundle
+   * @param lastKnownStart in case the app start is too long, resets the app start timestamp to this
+   *     value
+   */
+  public void updateAppStartType(final boolean hasBundle, final long lastKnownStart) {
```

Comment:
```suggestion

  /**
   * @param hasBundle true if the activity onCreate had a non-null bundle
   * @param startUptimeMillis in case the app start is too long, resets the app start timestamp to this
   *     value, retrieved via System.uptimeMillis()
   */
  public void updateAppStartType(final boolean hasBundle, final long startUptimeMillis) {
```
Just make it clear we want a time from System.uptimeMillis()